src/OpenFeatureSDK/Error/FeatureProviderException.cs

```diff
@@ -24,19 +23,7 @@ public class FeatureProviderException : Exception
         public FeatureProviderException(ErrorType errorType, string message = null, Exception innerException = null)
             : base(message, innerException)
         {

```

Comment:
In Java we use dedicated exception types for each error code. I'm not saying that's better, just an option to consider.

This might actually be easier for some providers :shrug: